### PR TITLE
Add json-ld context redirect for LDES

### PR DESCRIPTION
**Pull request by brechtvdv**

### CODE REVIEW

**Review by pietercolpaert — CHANGES_REQUESTED** [review]
I wouldn’t refer it to the jsdelivr anymore. This was a historic decision as github used to not support .jsonld and .ttl files properly. Can you change it to a subpath of https://semiceu.github.io/LinkedDataEventStreams/ and also change line 9 while on it?

File: ldes/.htaccess [hunk 1]
```diff
@@ -3,4 +3,7 @@ Header set Access-Control-Allow-Headers DNT,X-Mx-ReqToken,Keep-Alive,User-Agent,
 Options +FollowSymLinks
 RewriteEngine on
 RewriteRule ^specification$ https://semiceu.github.io/LinkedDataEventStreams/ [R=303,L]
-RewriteRule ^.*$ https://cdn.jsdelivr.net/gh/semiceu/LinkedDataEventStreams@master/vocabulary.ttl [R=303,L]
+
+RewriteRule ^context(.jsonld)?$ https://semiceu.github.io/LinkedDataEventStreams/context.jsonld [R=303,L]
```
Comment (dgarijo) [inline ldes/.htaccess:7]: this link is a 404

**Comment by dgarijo** [comment]
@pietercolpaert since you did an oiginal review, please re-review the changes. Thanks!

**Comment by pietercolpaert** [comment]
Changes look good. 404 is accurate: the pull request on the context.jsonld in LDES will be merged later

**Pull request merged**